x-pack/plugins/security_solution/public/common/components/event_details/table/prevalence_cell.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiLoadingSpinner } from '@elastic/eui';
+
+import { AlertSummaryRow } from '../helpers';
+import { useAlertPrevalence } from '../../../containers/alerts/use_alert_prevalence';
+
+const PrevalenceCell = React.memo<AlertSummaryRow['description']>(({ data, values }) => {
+  const { loading, count, error } = useAlertPrevalence({
```

Comment:
A benefit to making  a `PrevalenceCell` "own" it's IO for fetching the count via this hook is it's easy to add many `PrevalenceCell`s to a view, like a table view.

One side effect of this approach is a separate aggregation request will be made to Elasticsearch for every count displayed on the screen. For example, there are four fields in screenshot below:

![four_fields](https://user-images.githubusercontent.com/4459398/158438609-ce3ceca7-2613-425e-80d8-912b34bd61b6.png)

Four separate aggregation requests to Elasticsearch are made for each of the fields above, per the screenshot from Chrome dev tools below:

![network_requests](https://user-images.githubusercontent.com/4459398/158438737-e6fe9199-0235-42af-a489-e96ea345f835.png)

Should `PrevalenceCell` "own" it's IO for fetching the count, considering it requires a separate request for every field?

Reply:
@janmonschke we may want to revisit this on a follow up PR. I think moving it out and doing only one query makes more sense and _should_ be more performant.

Reply:
Yeah, I was thinking about this as well. I wasn't sure how much of a performance hit many of these requests would be. Given that we're also thinking about splitting up array values into separate rows, I'll likely have a look at that later. Thanks for the feedback!